coinjoins/README.md

```diff
@@ -0,0 +1,79 @@
+# Coinjoins & Privacy UX Primer
+
+- Bitcoin is not anonymous.
+- Consider each address you generate as a new identity.
+- Coin has history
+- Histories can become linked
+- Public ledger
+- Anyone can lookup
+- New users may not be aware
+
+## Address Reuse
+
+Use a [hierarchical deterministic wallet](https://en.bitcoin.it/wiki/Deterministic_wallet) in order to avoid reusing an identity.
+
+Practicality
+
+- How do you adverise an address to be paid in a video?
+
+## Wallet Fingerprinting
+
+The way the transaction is constructed can give chain analysts an idea of which wallet was used.
+
+1. Change is sometimes the last output
+   1. Solutions
+      1. Lexicographical Ordering
+2. If the timelock in the transaction is a specific amount
+
+## Routing Transaction
+
+- Should use Tor
```

Comment:
joinmarket, electrum personal server and wasabi support broadcasting by connecting to a random node over tor.

there are also well known broadcast services, such as blockstream.

more details on [the section on tor broadcasting on the bitcoin wiki Privacy page](https://en.bitcoin.it/wiki/Privacy#Tor_and_tor_broadcasting)

note that broadcasting with bitcoin core is potentially problematic for privacy, especially before the recent rebroadcast improvements.